dev-lang/vala/vala-0.40.1.recipe

```diff
@@ -0,0 +1,121 @@
+SUMMARY="An object-oriented programming language that uses the GObject system"
+DESCRIPTION="Vala is a new programming language that aims to bring modern \
+programming language features to GNOME developers without imposing any \
+additional runtime requirements and without using a different ABI \
+compared to applications and libraries written in C."
+HOMEPAGE="https://wiki.gnome.org/Projects/Vala"
+COPYRIGHT="2006-2018 Jürg Billeter
+	2006-2018 Raffaele Sandrini"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/vala/${portVersion%.*}/vala-$portVersion.tar.xz"
+CHECKSUM_SHA256="79b544305640b00010008cad0df1a1742363f636e4a7e3d410f651edc728f605"
+SOURCE_DIR="vala-$portVersion"
+PATCHES="vala-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
```

Comment:
```diff
-portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
```

Reply:
Will look into it :+1:

Reply:
Should I still drop portVersionCompat also?

Reply:
Yep, we don't need it.
This is because it is better to declare the dependencies on lib versions rather than on package versions.

Reply:
Bumped :)